### PR TITLE
Update Kallisto to 0.43.0 (resolves #224)

### LINKED ISSUE #224: need new kallisto version for rnaseq single cell
The single cell kallisto work needs version 0.43

### PR DESCRIPTION
**Pull request by tpesout**
'make build' and 'make test' both work and the version looks right.  I haven't run 'make push' yet, pending this PR.

@jvivian you said maybe 'wrapper.sh' wasn't needed (or should be removed?).  It's still the entry point, so let me know if that's wrong

### CODE REVIEW

File: kallisto/Makefile [hunk 1]
```diff
@@ -2,14 +2,14 @@
 build_output = runtime/kallisto
 runtime_fullpath = $(realpath runtime)
 build_tool = runtime-container.DONE
-git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../kallisto | cut -f1 -d " ")
+git_commit = a231803
```
Comment (jvivian) [inline kallisto/Makefile]: This is a bit confusing, but revert to the previous line. The git commit refers to _this repos commit hash_ not the commit of the tool you're pulling from.

**Review by jvivian — CHANGES_REQUESTED** [review]
Looks good, just a couple minor changes.  

PR title should end with *(resolves #224)*. 

Some of the advice is specific to the Toil repo, but this is a general guide we try to follow for PRs: https://github.com/BD2KGenomics/toil/wiki/Contributing

**Comment by tpesout** [comment]
@jvivian fixed it

**Comment by jvivian** [comment]
Thanks @tpesout — just waiting on tests.

**Review by jvivian — APPROVED** [review]

**Pull request merged**